test/e2e/framework.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Antrea Authors.
+// Copyright 2023 Antrea Authors.
```

Comment:
I think we don't need to update this when editing an existing file

Reply:
ack